### ISSUE TITLE
[REVIEW] Not able to run staticOptimization example for Python

### ISSUE THREAD

**Issue by abhishektha** [pyomeca/biorbd#172]
With the latest version from conda-forge I get the error:
```
Traceback (most recent call last):
  File "staticOptimization.py", line 38, in <module>
    optim = biorbd.StaticOptimization(model, Q, Qdot, Tau)
AttributeError: module 'biorbd' has no attribute 'StaticOptimization'
```

when running the staticOptimization.py Python example. I'm assuming the conda-forge version is using Eigen, which means this should be possible?

for JOSS Review https://github.com/openjournals/joss-reviews/issues/2562

**Comment by pariterre** [comment]
Hi @abhishektha 
Are you on Windows? If so, because of conflicting version of VS of Ipopt on conda-forge, I am unable to compile with the StaticOptimization module for that particular OS (it causes a linking error) for anaconda using conda-forge. 

Therefore, in order to use the StaticOptimization module on Windows, one must compile it by themselves. 

If you need, I could compile a version and add it to a self hosted channel on anaconda!

**Comment by abhishektha** [comment]
Yes this was on Windows! I don't think you need a new version on Anaconda for the paper submission, but maybe just put a note in the documentation about it (unless I missed it)!

I'll be trying the self-compiled versions later today!

**Comment by pariterre** [comment]
@abhishektha 
Yep, good idea! It is now done thanks to #176
